File: pgm/ABC_C#/ABC_156_B.cs
using System;

class Program
{
    static void Main()
    {
        String[] arr = Console.ReadLine().Split();
        int N = int.Parse(arr[0]), K = int.Parse(arr[1]);
        int cnt = 1;
        while (Math.Pow(K, cnt) <= N) cnt++;
        Console.WriteLine(cnt);
    }
}

File: pgm/ABC_Visual_Basic/ABC_156_B.vb
Class Main
    Public Shared Sub Main()
        Dim N As Integer
        Dim K As Integer
        Dim arr() As String
        arr = Split(Console.ReadLine())
        N = arr(0) : K = arr(1)
        Dim cnt As Integer = 1
        While K ^ cnt <= N
            cnt += 1
        End While
        Console.WriteLine(cnt)
    End Sub
End Class
